travis-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+dotnet test -c Debug -f netcoreapp1.0 MoreLinq.Test
+RID=`dotnet --info | grep RID | sed -E 's/ *RID: *(.+)/\1/'`
```

Comment:
I could switch it to iterate over all values:

```sh
for dir in MoreLinq.Test/bin/Debug/net451/*/ ; do
    mono $dir/dotnet-test-nunit.exe $dir/MoreLinq.Test.dll
done
```

This would be future proof against building multiple times.